changelog/unreleased/pull-3038

```diff
@@ -0,0 +1,7 @@
+Enhancement: Specify a percentage when using --read-data-subset
```

Comment:
Suggestion: `Enhancement: Allow specifying percentage in check's --read-data-subset`

Reply:
Fixed in https://github.com/restic/restic/pull/3038/commits/4943b1ebda8ad41c5cd485e202336e780a1e8c77